asgal

```diff
@@ -517,6 +517,9 @@ def main():
 
     args = parser.parse_args()
 
+    #if len(args.sample1Path) == 1:
```

Comment:
Another option: using -s as a list containing one or more paths to reads; this if statement would be necessary in order to keep old code working in case of a single file.